src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -312,3 +313,35 @@ def read_operation_fluid_data(self) -> Dict[str, Any]:
             "api_gravity": Scalar(fluid_data["api_gravity"], FRACTION_UNIT),
             "gas_gravity": Scalar(fluid_data["gas_gravity"], FRACTION_UNIT),
         }
+
+    def read_output_curves(self) -> Dict[str, Array]:
+        """Get the results from the input file."""
+        result = self.input_content["operation"]["thermal_simulation"]["result"]["0"]
```

Comment:
Here you are expecting that it exist, I'd rather use get, and check if it exist.

Reply:
Indeed, I check now if I have result.